common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1144,6 +1144,13 @@
     ],
     "sqlState" : "42623"
   },
+  "DESCRIBE_JSON_NOT_EXTENDED" : {
+    "message" : [
+      "DESCRIBE TABLE ... AS JSON only supported when [EXTENDED|FORMATTED] is specified.",
```

Comment:
this doesn't match the doc: https://github.com/apache/spark/pull/49139/files#diff-35ec15323468a71142a5eade7122a8855e190726d0ddde017783b0750745a526R41

only EXTENDED is allowed.